src/modules/blocks/event-category/block.js

```diff
@@ -2,50 +2,30 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Component } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './style.pcss';
 
-import {
-	TermsList,
-} from '@moderntribe/events/elements';
+import { TermsList } from '@moderntribe/events/elements';
 
 /**
  * Module Code
  */
 
-export default class EventCategory extends Component {
-	constructor() {
-		super( ...arguments );
-	}
-
-	render() {
-		return [
-			this.renderUI(),
-		];
-	}
-
-	renderUI() {
-		return (
-			<section key="event-category-box" className="tribe-editor__block">
-				<div className="tribe-editor__event-category">
-					{ this.renderList() }
-				</div>
-			</section>
-		);
-	}
-
-	renderList() {
-		return (
-			<TermsList
-				slug="tribe_events_cat"
-				label={ __( 'Event Category', 'the-events-calendar' ) }
-				renderEmpty={ __( 'Add Event Categories in document settings', 'the-events-calendar' ) }
-			/>
-		);
-	}
-}
-
+const EventCategory = ( props ) => {
+	return (
```

Comment:
Also, depending on the formatting rules on the project, we could just clean up the return

```suggestion
const EventCategory = ( props ) => (
      <section className="tribe-editor__block"> ...
```